packages/react-aria-components/src/Table.tsx

```diff
@@ -839,10 +839,11 @@ export const TableBody = /*#__PURE__*/ createBranchComponent('tablebody', <T ext
   let {dragAndDropHooks, dropState} = useContext(DragAndDropContext);
   let isDroppable = !!dragAndDropHooks?.useDroppableCollectionState && !dropState?.isDisabled;
   let isRootDropTarget = isDroppable && !!dropState && (dropState.isDropTarget({type: 'root'}) ?? false);
+  let isTableEmpty = collection.size === 0 && collection.rows.filter(row => row.type === 'loader').length === 0;
```

Comment:
If `collection.size` included loading rows I guess you wouldn't need this change?

Reply:
yep thats right